hack/logextractor.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+#
+# This script can be used to extract the failure cluster logs tarball from the PVC created for an install pod.
+#
+# Usage: ./logextractor.sh [CLUSTER_DEPLOYMENT_NAME] [DEST_DIR]
+#
+
+set -e
+
+POD_NAME="hive-logextractor"
+PVC_NAME=$(oc get pvc -l hive.openshift.io/cluster-deployment-name=$1 -o jsonpath="{.items[0].metadata.name}")
```

Comment:
For cleaner output in the case where the PVC does not exist, could I suggest the following. I think you need to get rid of the `set -e` for this to work, though you may want to do that anyway.

```
PVC_NAME=$(oc get pvc -l hive.openshift.io/cluster-deployment-name=$1 -o jsonpath="{.items[0].metadata.name}" 2> /dev/null)
if [ -z "$PVC_NAME" ]; then { echo "Could not find PVC for ClusterDeployment $1"; exit 1 }; fi
```
Otherwise, the script will put out esoteric output which may make it difficult to understand what went wrong.
```
error: error executing jsonpath "{.items[0].metadata.name}": Error executing template: array index out of bounds: index 0, length 0. Printing more information for debugging the template:
	template was:
		{.items[0].metadata.name}
	object given to jsonpath engine was:
		map[string]interface {}{"kind":"List", "apiVersion":"v1", "metadata":map[string]interface {}{"selfLink":"", "resourceVersion":""}, "items":[]interface {}{}}
```